app/styles/_helpers.scss

```diff
@@ -266,9 +266,56 @@ time {
     margin-bottom: auto !important;
 }
 
+
 // flex
 .sd-display-flex-column {
     display: flex;
     flex-direction: column;
     overflow: auto;
-}
\ No newline at end of file
+}
+
+
+// Common margin overrides
+// --------------------------------------------------
+
+// top
+.sd-margin-t--1 {
```

Comment:
What kind of design are you doing so you need such rules? :)

Reply:
Overrides for certain elements, to avoid creating to much modifiers for each element.

Reply:
Can you show an example of a design where you would put the override otherwise?

Reply:
Usage should be restricted, I prefer modifiers, but for certain elements we end up needing too much options. This is for such cases, I would avoid using it to much